src/dynamo0p3_builtins_mod.f90

```diff
@@ -175,22 +226,34 @@ module dynamo0p3_builtins_mod
   !> field3 = field1 + field2
   type, public, extends(kernel_type) :: plus_fields
      private
-     type(arg_type) :: meta_args(3) = (/                               &
-          arg_type(GH_FIELD,  GH_READ, ANY_SPACE_1),                   &
-          arg_type(GH_FIELD,  GH_READ, ANY_SPACE_1),                   &
-          arg_type(GH_FIELD, GH_WRITE, ANY_SPACE_1)                    &
+     type(arg_type) :: meta_args(3) = (/                              &
+          arg_type(GH_FIELD,  GH_READ, ANY_SPACE_1),                  &
+          arg_type(GH_FIELD,  GH_READ, ANY_SPACE_1),                  &
+          arg_type(GH_FIELD, GH_WRITE, ANY_SPACE_1)                   &
           /)
      integer :: iterates_over = DOFS
    contains
      procedure, nopass :: plus_fields_code
   end type plus_fields
 
+  !> field1 =  field1 ** ascalar
+  type, public, extends(kernel_type) :: raise_field
+     private
+     type(arg_type) :: meta_args(2) = (/                              &
+          arg_type(GH_FIELD, GH_INC, ANY_SPACE_1),                    &
+          arg_type(GH_REAL,  GH_READ)                                 &
```

Comment:
You missed one :-). I've modified it here https://github.com/stfc/PSyclone/pull/29/commits/977056eb2ed261134c224406da413ea0a706bf96.